packages/builder/src/actions/projects.ts

```diff
@@ -8,7 +8,7 @@ import { global } from "../global";
 import { RootState } from "../reducers";
 import { AppStatus, Application, ProjectStats } from "../reducers/projects";
 import { ProjectEvents, ProjectEventsMap } from "../types";
-import { graphqlFetch } from "../utils/graphql";
+import graphqlFetch from "../utils/graphql";
```

Comment:
Mostly personal preference, but I tend to always use named imports, because with default imports the name is up to the developer who writes the import, so there's a chance that different developers might pick different names in different files for the same imported thing. Could happen with named imports, too, but it takes an extra step so it's a little less likely.

Reply:
I do the same and agree, however there is a lint enabled (only in Builder) to prefer default exports when there's only a single export (that I don't think is a good idea :D ) - I'd say let's disable the lint globally - it's being eslint-ignored in other places of the project anyway.

Reply:
- disabled the lint rule and changed back to named export